bson/_cbsonmodule.c

```diff
@@ -1139,6 +1139,7 @@ static int _write_element_to_buffer(PyObject* self, buffer_t buffer,
         *(pymongo_buffer_get_buffer(buffer) + type_byte) = 0x03;
         return write_dict(self, buffer, value, check_keys, options, 0);
     } else if (PyList_Check(value) || PyTuple_Check(value)) {
+        int is_list = PyList_Check(value);
```

Comment:
this could be combined into the `if` above so that we only need to call PyList_Check once, not twice.

Reply:
done!